tray/monitor.go

```diff
@@ -177,9 +177,9 @@ func (ti *Instance) updateSettings() bool {
 
 func (ti *Instance) updateAccountInfo() bool {
 	changed := false
-	loggedIn := false
-	vpnActive := false
-	accountName := ""
+	loggedIn := ti.state.loggedIn
+	vpnActive := ti.state.vpnActive
+	accountName := ti.state.accountName
```

Comment:
Keep previous values - so, that subsequent request failure would not affect state